spryker/usr/local/share/env/60-framework

```diff
@@ -4,3 +4,8 @@
 export ELASTICSEARCH_HOST_PORT=${ELASTICSEARCH_PORT}
 export REDIS_HOST_PORT=${REDIS_PORT}
 export MAILCATCHER_HOST_PORT=${MAILCATCHER_PORT}
+
+# Spryker build as development
+export BUILD_DEVELOPMENT_MODE=${BUILD_DEVELOPMENT_MODE:-false}
+export BUILD_APPLICATION_ENV=${BUILD_APPLICATION_ENV:-development}
```

Comment:
we set symfony env to prod by default, shouldn't we also do the same for spryker?